File: abp/src/Wallee.Boc.Vote.Application.Contracts/Appraisements/AppraisementCreateDto.cs
using System;

namespace Wallee.Boc.Vote.Appraisements
{
    public class AppraisementCreateDto
    {
        public string Name { get; set; } = null!;
        public string Description { get; set; } = null!;

        public EvaluationCategory Category { get; set; }

        public DateTime Start { get; set; }
        public DateTime End { get; set; }
    }
}

File: vote_vite_mobile_client/src/openapi/models/AppraisementCreateDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { EvaluationCategory } from './EvaluationCategory';

export type AppraisementCreateDto = {
    name?: string | null;
    description?: string | null;
    category?: EvaluationCategory;
    start?: string;
    end?: string;
};
